packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -225,6 +226,18 @@ module.exports = {
     ],
   },
   plugins: [
+    // Lint CSS with stylelint
+    new StyleLintPlugin({
+      // @remove-on-eject-begin
+      config: {
+        extends: 'stylelint-config-react-app',
+      },
+      configBasedir: paths.ownPath,
+      // @remove-on-eject-end
+      allowEmptyInput: true,
```

Comment:
It seems `allowEmptyInput` have been removed.  
https://github.com/stylelint/stylelint/pull/2464